cmd/cheqd-noded/cmd/debug_extensions.go

```diff
@@ -42,7 +42,6 @@ func ed25519RandomCmd() *cobra.Command {
 
 			keyInfo := struct {
 				PubKeyBase64    	string `json:"pub_key_base_64"`
-				PubKeyMultibase58   string `json:"pub_key_multibase_58"`
```

Comment:
Isn't this used in our cheqd CLI identity instructions?

Reply:
No, another command is used to transcode `base64` -> `multibase58`. This is the empty field that we saw during our session.